text/0000-fabric-private-chaincode-lite.md

```diff
@@ -55,9 +48,11 @@ Such data includes secret cryptographic keys, which the chaincode uses to secure
     and also still mention all possible use-cases. 
     The distinction is done later in the architecture/design section
 -->
+## Enable new use-cases with strong privacy requrements
```

Comment:
```suggestion
## Enable new use-cases with strong privacy requirements
```

Reply:
thanks for catching that. I actually saw that and also changed the title to all capitals, somehow i must have forgot to save the corresponding change or alike ...